pkg/syncer/metadatasyncer.go

```diff
@@ -127,6 +129,7 @@ func (metadataSyncer *MetadataSyncInformer) Init() error {
 	}()
 
 	stopFullSync := make(chan bool, 1)
+	go volumes.ClearTaskInfoObjects()
```

Comment:
Now that syncer is no longer caching tasks, as it is only dealing with static volume creation, we can skip running this ClearTaskInfoObjects routine.

Reply:
Correct. +1